checker/raw_result.go

```diff
@@ -97,12 +110,43 @@ type Issue struct {
 	// TODO: add fields, e.g., state=[opened|closed]
 }
 
+// Commit represents a commit.
+type Commit struct {
+	// Note: SHA is not directly accessible from a pull request.
+	// TODO:SHA          string
+	Committer User
+	// Note: not all commits are reviewed.
+	// `nil` indicate no reviews.
+	Review *Review
+}
+
 // MergeRequest represents a merge request.
 type MergeRequest struct {
-	URL string
+	Number int
```

Comment:
Maybe `ID` instead of `Number`?

Reply:
PRs have both an `ID` and a `Number`,and those are different things. The `Number` is what we use in the UX, so I think that's what we want. Agreed?